build.rs

```diff
@@ -1,6 +1,8 @@
 use std::{env, fs, path::PathBuf};
 
 fn main() {
+    println!("cargo:rustc-cfg=const_fn_has_build_script");
```

Comment:
This is like a marker that means "build-script was run". If the build-script was run, but the output of the build-script is broken, you should get an error.